src/SDK.ts

```diff
@@ -955,4 +956,38 @@ export default class SDK implements ISDK {
       throw error;
     }
   }
+  
+  /** Send typing indicator
+   * @param requestId RequestId of the omnichannel session.
+   */
+  public async sendTypingIndicator(requestId:string): Promise<void> {
```

Comment:
I'd also add a guard to make sure this only works for v2

```
if (this.liveChatVersion !== LiveChatVersion.V2) { return new Error('Only supported on v2') }
```

Reply:
added this condition. THanks!